doc/about.rst

```diff
@@ -176,6 +176,8 @@ program.
 - 2015 - `Raghav RV <https://github.com/raghavrv>`_, Wei Xue
 - 2016 - `Nelson Liu <http://nelsonliu.me>`_, `YenChen Lin <https://yclin.me/>`_
 
+.. _Vlad Niculae: https://vene.ro/
```

Comment:
Any idea why sphinx does not find the identical ref in `doc/whats_new/_contributors.rst` ?
It feels a bit weird like that. I would rather make sphinx find the ref, or write directly:
```
`Vlad Niculae <https://vene.ro/>`_
```